### PR TITLE
[WIP][8.0][BR000973][T22092]Refactor elico_project into several distinct modules[project_task_type]

### PR DESCRIPTION
**Pull request by dongshukai**
This PR is about the module project_task_type

### CODE REVIEW

File: project_task_type/README.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+To install this module, you need to:
+
+ * have basic modules installed (project, project_issue)
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/Elico-Corp/odoo/issues>`_.
```
Comment (elicoidal) [inline project_task_type/README.rst]: see https://github.com/Elico-Corp/odoo-addons/pull/171/files

File: project_task_type/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# © 2015-2017 Elico corp (http://www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Project Task Type',
+    'version': '8.0.1.0.1',
```
Comment (elicoidal) [inline project_task_type/__openerp__.py]: 8.0.1.0.0

File: project_task_type/project.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+# © 2015-2017 Elico corp (http://www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+
+class ProjectTask(models.Model):
+    _inherit = 'project.task'
+
+    type = fields.Selection(
```
Comment (elicoidal) [inline project_task_type/project.py:11]: I think that is obsolete now
@victormartinelicocorp

**Review by elicoidal — COMMENTED** [review]
I think we do not need this module anymore
cc @victormartinelicocorp @seb-elico

**Comment by dongshukai** [comment]
@Elico-Corp/odoo-reviewers 
pls check

**Comment by dongshukai** [comment]
accidebt click

File: project_task_type/project.py (hunk 3 above)
Comment (martintorresvm) [reply project_task_type/project.py:11]: @elicoidal indeed already we have the categories.

**Comment by elicoidal** [comment]
obsolete with the BR system

**Pull request closed without merging**